docs/topics/OBST-wf-swarm-crud-deploy-to-openshift-maven.adoc

```diff
@@ -0,0 +1,34 @@
+= Deploy the Database Application and CRUD API to OpenShift
```

Comment:
In this section, you are using titled sections to represent discrete steps. I think you can accomplish this with just steps. For example:

```
. Start the database by creating a new PostgreSQL database application.
+
[source,bash,option='nowrap']
--
 oc new-app -e POSTGRESQL_USER=swarm -ePOSTGRESQL_PASSWORD=password -ePOSTGRESQL_DATABASE=fruits openshift/postgresql-92-centos7 --name=my-database
--

. Build and deploy the CRUD HTTP application.
+
[source,bash]
--
 mvn clean fabric8:deploy -Popenshift -Dskiptests
--

. Wait for the deployment to complete.
...
```